src/BookingInfo.php

```diff
@@ -151,35 +131,28 @@ public function sameAs(BookingInfo $otherBookingInfo): bool
 
     public static function fromUdb3ModelBookingInfo(Udb3ModelBookingInfo $udb3ModelBookingInfo): BookingInfo
     {
-        $website = null;
-        $phone = null;
-        $email = null;
-        $availabilityStarts = null;
-        $availabilityEnds = null;
-
-        if ($udb3ModelWebsite = $udb3ModelBookingInfo->getWebsite()) {
-            $website = $udb3ModelWebsite;
-        }
+        return new self(
```

Comment:
this is so much nicer!

Reply:
Finally this class gets removed because it's converting the new model to the legacy model which now has the same type of properties 😏

Reply:
See https://github.com/cultuurnet/udb3-backend/pull/1907

BTW, there is a story in these pull requests, from bottom to top  😊. Small steps of making the old legacy model look and behave like the new model.